### PR TITLE
3.get/api/topics complete

### PR DESCRIPTION
**Pull request by ellamarym**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+const express = require("express");
+const { getTopics } = require("./controllers/app.controller");
+const app = express();
+app.use(express.json());
```
Comment (Geo-Beard) [inline app.js]: Don't need this at the moment - remove then add when you need it in future ticket :)

Comment (ellamarym) [reply app.js]: i dont need to create the app?

Comment (ellamarym) [reply app.js]: or just the express.json()

File: __tests__/app.test.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+const request = require("supertest");
+const app = require('../app');
+const db = require('../db/connection');
+const testData = require('../db/data/test-data/index')
+const seed = require('../db/seeds/seed')
+beforeEach(() => seed( testData ));
+afterAll(() => {
+    if (db.end) db.end();
```
Comment (Geo-Beard) [inline __tests__/app.test.js]: Can change this to a single line - afterAll(() => db.end());

**Comment by Geo-Beard** [comment]
Looks good! Made a few comments, if you make those changes you can merge without opening another PR :)

File: app.js (hunk 1 above)
Comment (Geo-Beard) [reply app.js]: Just the app.use(express.json()) - you don't need that line til later

**Pull request merged**